tests/getting-started-complete-test.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
+    tags: ['getting-started-complete-test'],
+    'Smoke test getting-started-complete' : function (client) {
+      client
+        .url('http://localhost:3000')
+        .pause(1000);
```

Comment:
Could you replace this `pause` here with a `waitForElementVisible` call instead? I don't recall needing to have this when I was writing my tests so I'm curious why it's here.